test/integration/rules/runner.js

```diff
@@ -102,12 +102,27 @@
 							axe.run(
 								fixture,
 								{
+									/**
+									 * The debug flag helps log errors in a fairly detailed fashion,
+									 * when tests fail in webdriver
+									 */
+									debug: true,
```

Comment:
This prevents errors from getting passed to `results.incomplete` and for axe.run to throw. We can do one or the other solution (whichever gives the clearer error), but not both.

Reply:
That is not true.

If any of the assertions fail (with or without this flag), the current set-up does not give a detailed logging on which part of the code failed. Enabling `debug`, prints the line number in question for the failure with respect to the library, and not just the chai assertion.

Also, if an assertion fails I would expect to throw (for example like below)
![image](https://user-images.githubusercontent.com/20978252/43828829-7d7f1ae6-9af5-11e8-819d-0fbc007387a0.png)

I have tested this with both enabling/ disabling the debug flag and is working as expected.